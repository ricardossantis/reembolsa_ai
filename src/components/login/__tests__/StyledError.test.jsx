import React from "react";
import renderer from "react-test-renderer";
import StyledError from "../styled-error";

describe("Render Test", () => {
  it("renders StyledError", () => {
    const component = renderer.create(<StyledError/>).toJSON();
    expect(component).toMatchSnapshot();
  });
});
