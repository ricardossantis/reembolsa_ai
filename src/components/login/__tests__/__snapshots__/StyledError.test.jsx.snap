// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Test renders StyledError 1`] = `
<p
  className="sc-bdnylx ekZepH"
/>
`;
